util/Commands.re

```diff
@@ -34,7 +34,7 @@ let execFull = (~input=?, ~pwd=?, ~env=Unix.environment(), cmd) => {
   | None => ()
   | Some(text) => output_string(cmd_in, text)
   };
-  close_out(cmd_in);
```

Comment:
`close_out` would trigger a `Sys_error("Broken pipe")` exception now that we ignore the `SIGPIPE` signal. Instead of catching and handling it, we simply use `close_out_noerr` which ignores all errors